pages/index.py

```diff
@@ -27,35 +27,58 @@ def slider_item(title, text, active=""):
             slider_item(_("Share, Chat, Browse. Anonymously."), _("""
 Freenet is a platform for censorship-resistant communication and publishing.
 It helps you to remain anonymous, and communicate without fear.
-"""), "active"),
+""") + "\n\n" + """<span class="hidden-xs" markdown="1">""" + _("""
```

Comment:
it isn’t needed … it would only be needed for some markdown libraries if the <span> were inside the string parsed as markdown. I forgot removing it after refactoring. sorry